package.json

```diff
@@ -25,11 +24,13 @@
     "format": "prettier --write . && npm run lint -- --fix",
     "build": "npm-run-all build:**",
     "build:clean": "del-cli lib",
-    "build:esm": "babel src --out-dir lib/esm --extensions \".ts\" --source-maps --config-file ./babel.config.json",
-    "build:umd": "rollup --config rollup.config.js && cpy tools/cjs/package.json lib/umd --flat",
+    "build:esm": "babel src --out-dir lib/esm --extensions \".ts\" --source-maps --config-file ./babel.config.json && npm run cpy:esm",
+    "build:umd": "rollup --config rollup.config.mjs && npm run cpy:cjs",
     "build:types": "tsc --project tsconfig-types.json",
     "build:validate": "jest --rootDir='test-lib' --runInBand",
     "build-and-test": "npm test && npm run lint && npm run build",
+    "cpy:cjs": "cpy tools/cjs/package.json lib/umd --flat",
```

Comment:
Can you rename this script to `build:umd:cpy`, move it right under `build:umd`, and remove the `&& ...` from `build:umd`? That will simplify the scripts a bit.

And the same with `cpy:esm`?